features/url-rewriters/akamai.md

```diff
@@ -0,0 +1,17 @@
+# Akamai URL-rewriter
+
+ImageKit can translate transformations written in Akamai's syntax to ImageKit's syntax. Learn how to use a URL-rewriter on [this page](/features/url-rewriters).
+
+{% hint style="info" %}
+**Beta feature**\
+This feature is currently in beta. As a result, some transformations may work differently than expected.
+{% endhint %}
+
+## Supported Transformations
+
+The following Akamai transformations are supported for translation:
+
+| Akamai transform | Example translation | Remarks |
+| - | - | - |
+| <p>width</p> | <ul> <li> `imwidth=100` => `w-100`</li> </ul> | <ul>Final output from ImageKit may vary from output from Akamai</ul> |
```

Comment:
It can be written as "The final result produced by ImageKit could differ from the result obtained from Akamai." I guess imgix se copy hua hai ye, vaha bhi change karna chahiye.

Though main point is ki hai kyu ye remark and fir kese confidence aayega user ko. Agar difference sirf kuch pixels ka hi hai to why even mention it?

Reply:
Removed it.